### PR TITLE
Add pointerEvents: none to left/right sides.

### PR DESCRIPTION
**Pull request by agamemnus**
Without this change, if the zIndex of the waves is higher than the zIndex of the regions, the waves will stand out better but block the resize controls.

(Figured out how to do two files at once.)

### CODE REVIEW

**Comment by katspaugh** [comment]
Nice idea!

**Comment by thijstriemstra** [comment]
@agamemnus can you also add this to the documentation?

**Comment by agamemnus** [comment]
Ok, in a bit. By the way, what is the plan for merging changes into next?
Are you just going to do a bunch of monster merges?

**Comment by agamemnus** [comment]
I'm not sure where I'd add it since it's not an option.

File: src/drawer.multicanvas.js [hunk 1]
```diff
@@ -72,11 +73,12 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
         entry.wave = this.wrapper.appendChild(
             this.style(document.createElement('canvas'), {
                 position: 'absolute',
-                zIndex: 1,
+                zIndex: 2,
```
Comment (glebm) [inline src/drawer.multicanvas.js]: This makes it so that this canvas is above the progress wave, hiding it. See #1136

**Pull request merged**